en_US.json

```diff
@@ -1,4 +1,6 @@
 {
+    "Player Database": "Player Database",
```

Comment:
Remove this entry, I'll work on translating the rest of the menu later on.

Reply:
already removed, along with some more useless ones lol